main.go

```diff
@@ -103,6 +103,7 @@ func mainE(ctx context.Context) error {
 
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
+	daemonCommand.PersistentFlags().String(f.Service.NamespaceAuth.ViewAllTargetGroup, "", "Identity provider group, allowed to view all resources in tenant namespace.")
```

Comment:
Can't we predefine this name and have it identical in all CPs?

Reply:
It is group name which comes from customers identity provider. We can't predefine it

Reply:
Ack.

Then how about this to make it more expressive:
```suggestion
	daemonCommand.PersistentFlags().String(f.Service.NamespaceAuth.Role.ViewAll.TargetGroup, "", "Identity provider group, allowed to view all resources in tenant namespace.")
```

Reply:
I thought about this. But I don't expect there many options for configuration. For now only three planned: view all, tenant admin and app admin. I kind of don't want to make a separate structure for every option (adding ~9 fields into every installation config file instead of 3).
Also `role` itself doesn't have group. It is only binding